libs/@local/repo-chores/scripts/create-block.ts

```diff
@@ -0,0 +1,77 @@
+import path from "node:path";
+
+import { JsonObject } from "@blockprotocol/core";
+import execa from "execa";
+import fs from "fs-extra";
+
+import { monorepoRootDirPath } from "./shared/monorepo";
+
+const script = async () => {
+  const args = process.argv.slice(2);
+
+  const blockName = args[0];
+
+  if (!blockName) {
+    console.log(
+      "Please provide a block name as the first argument, e.g. yarn create-block my-block",
+    );
+    process.exit();
+  }
+
+  const blocksFolder = path.join(monorepoRootDirPath, "blocks");
+
+  console.log("************ Creating block ************");
+
+  await execa("npx", ["create-block-app@latest", ...args], {
+    cwd: path.join(monorepoRootDirPath, "blocks"),
+    stdout: "inherit",
+  });
+
+  console.log("********* Configuring for repo *********");
+  const newBlockFolder = path.join(blocksFolder, blockName);
+
+  const packageJsonPath = path.join(newBlockFolder, "package.json");
+
+  const packageJsonString = fs.readFileSync(packageJsonPath, "utf-8");
+
+  const packageJson = JSON.parse(packageJsonString) as JsonObject;
+
+  packageJson.name = `@blocks/${blockName}`;
+  packageJson.license = "(MIT OR Apache-2.0)";
+  packageJson.author = "HASH";
```

Comment:
This doesn't seem to work for me, I've still got old fields within it:
```
  "license": "MIT",
  "author": "Alfred Mountfield",
```

Reply:
Oh oops, I did notice that then somehow forgot it

Reply:
Fixed in https://github.com/hashintel/hash/commit/725bd38352ad148282a61eb199430f3eb8518a87

Reply:
Cheers